src/js/containers/aboutTheData/AgenciesContainer.jsx

```diff
@@ -1,14 +1,17 @@
-import React, { useRef, useState } from "react";
+import React, { useEffect, useRef, useState } from "react";
```

Comment:
Looks like there are some prop issues and keys needed for the list elements.
<img width="409" alt="Screen Shot 2020-11-24 at 10 35 09 AM" src="https://user-images.githubusercontent.com/14301977/100115960-2ac15f80-2e41-11eb-93e8-297ed5816b6e.png">
<img width="408" alt="Screen Shot 2020-11-24 at 10 35 22 AM" src="https://user-images.githubusercontent.com/14301977/100115963-2b59f600-2e41-11eb-95d1-a00714c1ee1f.png">
<img width="402" alt="Screen Shot 2020-11-24 at 10 37 15 AM" src="https://user-images.githubusercontent.com/14301977/100115964-2b59f600-2e41-11eb-8461-43c0e20deb5d.png">